Update instructions to include more UI challenge and remove requirements confusion (#7)
ledn-reviewer/ledn-frontend-challenge: README.md

## Requirements

Your task is to design the foundation of a responsive user interface capable of identifying planets susceptible to uprisings and enabling proactive intervention. The objective is to build a UI that assists administrators in analyzing transactions across various planets. To meet this goal, adhere to the following guidelines:

1. **Summary**
  - The administrators would like to see a summary of all the planets of the galaxy.
  - From that list, they would like to be able to search for a planet by its name.

2. **Detail**
  - The administrators would like to access detailed information of the planet.
  - They would like to be able to filter the transactions by its currency type (GSC (Galactic Credit Standard) and ICS (Imperial Crown Standard)).

>   - They would like to be able to see each transaction in the different currencies of the galaxy according to a real-time rate.

Reviewer: ```suggestion
  - They would like to be able to see each transaction in each currencies (GSC and ISC) according to a real-time rate.
```

### Technical requirements
To ensure the success of your mission, adopt the following technical specifications:

* **Technologies & languages:**
  - Use `React Query` to execute all the requests to the endpoints.
  - Build the solution using `Typescript` and `React`.
  - In order to allow us to review a similar set of challenges, do not update, change or edit the mock server.

* **User interface:**
  - Organize the UI to the best of your knowledge and judgment.
  - In order to allow us to review a similar set of challenges, only use libraries already present in the package.json.

>   - In order to allow us to review HTML and CSS knowledge, the component to filter the transactions by its currency type should be coded from scratch.

Reviewer: Without using the UI libraries
